app/controllers/carts_controller.rb

```diff
@@ -0,0 +1,41 @@
+class CartsController < ApplicationController
+  def show
+    @order_items = current_order.order_items
+
+    calc_unit_total
+    calc_order_total
+  end
+
+  def calc_unit_total
```

Comment:
It seems like these two methods are being used to assist with the `show` method, so I think it would make sense to make them `private` since they are no accessed outside this controller